datalad/core/distributed/clone.py

```diff
@@ -571,16 +586,64 @@ def postclonecfg_annexdataset(ds, reckless, description=None):
     lgr.debug("Initializing annex repo at %s", ds.path)
     # Note, that we cannot enforce annex-init via AnnexRepo().
     # If such an instance already exists, its __init__ will not be executed.
-    # Therefore do quick test once we have an object and decide whether to call its _init().
+    # Therefore do quick test once we have an object and decide whether to call
+    # its _init().
     #
     # Additionally, call init if we need to add a description (see #1403),
     # since AnnexRepo.__init__ can only do it with create=True
     repo = AnnexRepo(ds.path, init=True)
     if not repo.is_initialized() or description:
         repo._init(description=description)
-    if reckless:
+    if reckless == 'auto':
         repo._run_annex_command('untrust', annex_options=['here'])
 
+    elif reckless == 'ephemeral':
+        # with ephemeral we declare 'here' as 'dead' right away, whenever
+        # we symlink origins annex. Because we want annex to copy to
+        # the ria remote to get availability info correct for an eventual
+        # git-push into the store
```

Comment:
I'm having a hard time parsing/understanding from "Because [...]" to here.  If it is meant to convey what is said in the commit message (re: not propagating back), I think that phrasing is clearer.